etc/picongpu/jureca-jsc/gpus_picongpu.profile.example

```diff
@@ -10,55 +10,65 @@ export MY_MAIL="someone@example.com"
 export MY_NAME="$(whoami) <$MY_MAIL>"
 
 # Project Information ######################################## (edit this line)
-#   - project account for computing time
-export proj=$(groups | awk '{print $5}')
+#   - project and account for allocation
+#   jutil user projects will return a table of project associations.
+#   Each row contains: project,unixgroup,PI-uid,project-type,budget-accounts
+#   We need the first and last entry.
+#   Here: select the last available project.
+export proj=$( jutil user projects --noheader | awk '{print $1}' | tail -n 1 )
+export account=$(jutil user projects -n | awk '{print $NF}' | tail -n 1)
 
 # Text Editor for Tools ###################################### (edit this line)
 #   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
 #export EDITOR="nano"
 
 # Set up environment, including $SCRATCH and $PROJECT
-jutil env activate -p $proj
+# Handle a case where the budgeting account is not set.
+if [ "$account" = "-" ]; then
+    jutil env activate --project $proj;
+else
+    jutil env activate --project $proj --budget $account
+fi
 
 # General modules #############################################################
 #
 module purge
-module load GCC/7.3.0
-module load CUDA/9.2.88
+module load Stages/2024
+module load GCC/12.3.0
+module load CUDA/12
 module load CMake
-module load MVAPICH2/2.3-GDR
-module load Python/3.6.6
-
-# Other Software ##############################################################
+module load OpenMPI/4.1.5
+module load Python/3.11.3
+module load ADIOS2/2.9.2
+module load HDF5/1.14.2
+module load Boost/1.82.0
+
+# necessary for evaluations (NumPy, SciPy, Matplotlib, SymPy, Pandas, IPython)
+module load SciPy-bundle/2023.07
+module load matplotlib/3.7.2
+module load h5py/3.9.0
+
+# Self-Build Software #########################################################
 #
-module load zlib/.1.2.11
-module load libpng/.1.6.35
-export CMAKE_PREFIX_PATH=$EBROOTZLIB:$EBROOTLIBPNG:$CMAKE_PREFIX_PATH
-
-PARTITION_LIB=$PROJECT/lib_gpus
-BOOST_ROOT=$PARTITION_LIB/boost
-export CMAKE_PREFIX_PATH=$BOOST_ROOT:$CMAKE_PREFIX_PATH
-export LD_LIBRARY_PATH=$BOOST_ROOT/lib:$LD_LIBRARY_PATH
-
-HDF5_ROOT=$PARTITION_LIB/hdf5
-export PATH=$HDF5_ROOT/bin:$PATH
-export CMAKE_PREFIX_PATH=$HDF5_ROOT:$CMAKE_PREFIX_PATH
-export LD_LIBRARY_PATH=$HDF5_ROOT/lib:$LD_LIBRARY_PATH
+# needs to be compiled by the user
+# Check the install script at
+# https://gist.github.com/steindev/5a2ab0c4957a1e57ed671ac198b491c6
```

Comment:
This is not up-to-date and should not be linked here. @PrometheusPi don't you have something recent?

Reply:
No, I never wrote a script for it - too much testing in the end. 
I would favor keeping the link as can be seen as a reference.

Reply:
Short *offline* discussion with @steindev. The gist is so outdated that we shoudl remove it.

Reply:
If you upload the script to your gist, that would be perfect. Then you could discard my last commit (to your pull request), add the link to your gist and squash everything to a single commit.